sonar-scanner-engine/src/main/java/org/sonar/batch/report/ScannerReportUtils.java

```diff
@@ -20,11 +20,11 @@
 package org.sonar.batch.report;
 
 import org.sonar.api.batch.sensor.highlighting.TypeOfText;
-import org.sonar.batch.protocol.Constants.HighlightingType;
+import org.sonar.scanner.protocol.Constants.HighlightingType;
 
-public class BatchReportUtils {
+public class ScannerReportUtils {
 
-  private BatchReportUtils() {
+  private ScannerReportUtils() {
   }
 
   public static HighlightingType toProtocolType(TypeOfText textType) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this method 9, down to the maximum allowed 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1142)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "toProtocolType" is 20 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)